### PR TITLE
[Server] Circle notifications partII: handle daily emails

### LINKED ISSUE #2548: [Server] Circle notifications
tasks
- [ ] Add handler of new circle notifications
- [x] Add API for circle notification settings
- [ ] Add notices into dailySummary email (follow logic of dailySummary email)

refs
- [doc](https://paper.dropbox.com/doc/Matters--BiiFkeFTRl0Vt5jY9wwDEAEzAg-jWhP7c3QBJKT4RT73xLab#:uid=858246946599872387512974&h2=2%EF%B8%8F%E2%83%A3-%E9%80%9A%E7%9F%A5%E8%BF%AD%E4%BB%A3%E6%96%B9%E6%A1%88)

design
- [figma](https://www.figma.com/file/oGew0i2s0N8IBm9tCpoLLI/%E8%BF%AD%E4%BB%A3%E8%A8%AD%E8%A8%88%E4%B8%AD%E7%B9%BC%E7%AB%99?node-id=4030%3A24449)

### PR DESCRIPTION
**Pull request by tx0c**
also add settings case for article_new_collected

resolves #2548

### CODE REVIEW

File: src/connectors/notificationService/mail/sendDailySummary.ts [hunk 1]
```diff
@@ -123,6 +205,20 @@ export const sendDailySummary = async ({
...
+            circle_new_unsubscriber,
+            circle_new_broadcast,
+            circle_new_discussion,
+
+            // for members in circle
+            in_circle_new_article,
+            in_circle_new_boradcast,
+            in_circle_new_boradcast_reply,
+            in_circle_new_discussion,
+            in_circle_new_discussion_reply,
```
Comment (tx0c) [inline src/connectors/notificationService/mail/sendDailySummary.ts:221]: to @ElkLeung here's new data for Daily Summary email design

File: src/types/user.ts [hunk 2]
```diff
@@ -365,9 +366,9 @@ export default /* GraphQL */ `
     circleNewFollower: Boolean!
     circleNewUnsubscriber: Boolean!
     circleNewDiscussion: Boolean!
-    circleMemberBoradcast: Boolean!
-    circleMemberNewDiscussion: Boolean!
-    circleMemberNewDiscussionReply: Boolean!
+    circleMemberBoradcast: Boolean! # deprecated
```
Comment (robertu7) [inline src/types/user.ts:369]: use directive `@deprecated(reason: "No longer in use")`?

**Comment by tx0c** [comment]
landed in a80c6586

File: src/types/user.ts (hunk 2 above)
Comment (tx0c) [reply src/types/user.ts:369]: resolved in a80c6586

**Pull request closed without merging**